.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
 
 script:
   - npm run build
-  - mkdir build && cp -r static templates parsapp.py database.py build
+  - mkdir build && cp -r static templates parsapp.py database.py reset_db.py build
 
 after_success:
-  - '[ "${TRAVIS_BRANCH}" = "master" ] && rsync -rq build/ $DEPLOY_USER@$DEPLOY_HOST:'
+  - '[ "${TRAVIS_BRANCH}" = "development" ] && rsync -rq build/ $DEPLOY_USER@$DEPLOY_HOST:$DEPLOY_PATH'
```

Comment:
change back to master, as now is ready to merge

Reply:
Travis ci is obsolete for the docker deployment. We can probably completely delete it.

Reply:
ok,  then remove it, it currently stalls the build on travis because it's asking for a password